pkgs/development/compilers/dtc/default.nix

```diff
@@ -14,44 +16,61 @@
 
 stdenv.mkDerivation rec {
   pname = "dtc";
-  version = "1.6.1";
+  version = "1.7.0";
 
   src = fetchgit {
     url = "https://git.kernel.org/pub/scm/utils/dtc/dtc.git";
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-gx9LG3U9etWhPxm7Ox7rOu9X5272qGeHqZtOe68zFs4=";
+    sha256 = "sha256-FMh3VvlY3fUK8fbd0M+aCmlUrmG9YegiOOQ7MOByffc=";
   };
 
   patches = [
-    # fix python 3.10 compatibility
-    # based on without requiring the setup.py rework
-    # https://git.kernel.org/pub/scm/utils/dtc/dtc.git/commit/?id=383e148b70a47ab15f97a19bb999d54f9c3e810f
-    ./python-3.10.patch
+    # meson: Fix cell overflow tests when running from meson
+    (fetchpatch {
+      url = "https://github.com/dgibson/dtc/commit/32174a66efa4ad19fc6a2a6422e4af2ae4f055cb.patch";
+      sha256 = "sha256-C7OzwY0zq+2CV3SB5unI7Ill2M3deF7FXeQE3B/Kx2s=";
+    })
 
-    # fix dtc static building
-    ./0001-Depend-on-.a-instead-of-.so-when-building-static.patch
+    # meson.build: bump version to 1.7.0
+    (fetchpatch {
+      url = "https://github.com/dgibson/dtc/commit/64a907f08b9bedd89833c1eee674148cff2343c6.patch";
+      sha256 = "sha256-p2KGS5GW+3uIPgXfuIx6aDC54csM+5FZDkK03t58AL8=";
+    })
+
+    # Fix version in libfdt/meson.build
+    (fetchpatch {
+      url = "https://github.com/dgibson/dtc/pull/95/commits/723545ebe9933b90ea58dc125e4987c6bcb04ade.patch";
```

Comment:
Why some are in /pull but others in /commit?

Reply:
This has not been merged; others are.

Reply:
When I enter 

https://github.com/dgibson/dtc/pull/95/commits/723545ebe9933b90ea58dc125e4987c6bcb04ade.patch

it redirects to

https://github.com/dgibson/dtc/commit/723545ebe9933b90ea58dc125e4987c6bcb04ade.patch

No real difference.

Reply:
Fixed.